### PR TITLE
offline first: add a hint that the help is also available for relaxed, in-app reading when being offline

### PR DESCRIPTION
**Pull request by r10s**
just came to my mind, pointing out a bit that the help is really available offline - which is not natural in maaaany other apps:

Screenshot 2023-04-06 at 12 56 08

### CODE REVIEW

File: en/help.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ header: Frequently Asked Questions
 
 ## What is Delta Chat?
 
+> In a hurry? You can read this help relaxed at anytime within the app, also without internet.
```
Comment (rosano) [inline en/help.md]:
```suggestion
> You can also read this help relaxed at anytime within the app, and it works without internet.
```

I would remove the 'hurry', and slightly reword.

**Review by rosano — APPROVED** [review]
I think it would be more helpful to move it before the table of contents so that scrolling isn't necessary.

**Comment by r10s** [comment]
> I think it would be more helpful to move it before the table of contents so that scrolling isn't necessary.

+1 - however, there was an issue with that,  but i also did not try hard. wanted some general feedback first, if the hint makes sense at all :)

File: en/help.md (hunk 1 above)
Comment (r10s) [reply en/help.md]: hm, the gist was also to be less formal than the rest of the page

Comment (rosano) [reply en/help.md]: Not a blocker

**Comment by r10s** [comment]
found a way to move it up, via strings-yaml-file - this has also the advantage that no adaptions are needed for the offline help creation

**Review by missytake — APPROVED** [review]
Love it! Was wondering about whether you want that sentence in the in-app-help as well :upside_down_face:

**Review by adbenitez — APPROVED** [review]

**Pull request merged**